internal/refreshtoken/refresh_token.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/hashicorp/boundary/internal/types/resource"
 )
 
+// UpdatedTimeBuffer is used to automatically adjust the updated
+// time of a refresh token to account for delays between database
+// transactions. The buffer corresponds to Postgres' default transaction timeout.
```

Comment:
> The buffer corresponds to Postgres' default transaction timeout.

I don't think this is correct. As far as I know, postgres does not have a transaction timeout. There is an `idle_in_transaction_session_timeout` that will terminate a session that has been idle for too long. And there is `statement_timeout` but the default is `0`, meaning no timeout.

Reply:
@mikemountain where did we source this from? I can't remember 😅

Reply:
@johanbrandhorst I can't seem to find where we did end up sourcing this from... but that being said, I think 30 seconds is still a reasonable buffer

Reply:
I'll replace the Postgres timeouts with a different blurb 👍🏻